lukejagg/2048
# Increase padding variable to 15

## Description
This PR increases the padding variable in the Game.java file to 15. The padding variable is used to set the padding around the cells in the game. Increasing this value creates more space between the cells, potentially improving the visual appearance of the game.

## Summary of Changes
- Modified the Game.java file to increase the padding variable from 5 to 15.
- Reviewed the methods that use the padding variable, including the draw() and start() methods, to ensure they still function correctly with the increased padding.
- Tested the game after making the change to ensure that the UI looks as expected and there are no unexpected side effects.

Fixes #1.

---
To checkout this PR branch, run the following command in your terminal:
```zsh
git checkout sweep/increase-padding_2
```

---

🎉 Latest improvements to Sweep:

* Use Sweep Map to break large issues into smaller sub-issues, perfect for large tasks like "Sweep (map): migrate from React class components to function components"
* Getting Sweep to format before committing! Check out [Sweep Sandbox Configs](https://docs.sweep.dev/config#sandbox) to set it up.
* We released a [demo of our chunker](https://huggingface.co/spaces/sweepai/chunker), where you can find the corresponding blog and code.


---

💡 To get Sweep to edit this pull request, you can:
* Leave a comment below to get Sweep to edit the entire PR
* Leave a comment in the code will only modify the file
* Edit the original issue to get Sweep to recreate the PR from scratch
